Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+
+            //string runfieldInRestSyntax = QCRestAPIConnect.ConvertResourceType(ALM_Common.DataContracts.ResourceType.TEST_RUN);
+            List<QTestApiModel.FieldResource> runfieldsCollection = fieldApi.GetFields((long)Convert.ToInt32(ALMCore.AlmConfig.ALMProjectKey), "test-runs");
+
+            fields.Append(AddFieldsValues(testSetfieldsCollection, "test-suites"));
+            fields.Append(AddFieldsValues(testCasefieldsCollection, "test-cases"));
+            //fields.Append(AddFieldsValues(designStepfieldsCollection, designStepfieldInRestSyntax));
+            //fields.Append(AddFieldsValues(testInstancefieldsCollection, testInstancefieldInRestSyntax));
+            //fields.Append(AddFieldsValues(designStepParamsfieldsCollection, designStepParamsfieldInRestSyntax));
+            fields.Append(AddFieldsValues(runfieldsCollection, "test-runs"));
+
+            return fields;
+        }
+        private static ObservableList<ExternalItemFieldBase> AddFieldsValues(List<QTestApiModel.FieldResource> testSetfieldsCollection, string testSetfieldInRestSyntax)
+        {
+            ObservableList<ExternalItemFieldBase> fields = new ObservableList<ExternalItemFieldBase>();
+
+            if ((testSetfieldsCollection != null) && (testSetfieldsCollection.Count > 0))
+            {
+                foreach (QTestApiModel.FieldResource field in testSetfieldsCollection)
+                {
+                    if (string.IsNullOrEmpty(field.Label)) continue;
+
+                    ExternalItemFieldBase itemfield = new ExternalItemFieldBase();
+                    itemfield.ID = field.OriginalName;
+                    itemfield.ExternalID = field.OriginalName;  // Temp ??? Check if ExternalID has other use in this case
+                    itemfield.Name = field.Label;
+                    bool isCheck;
+                    itemfield.Mandatory = bool.TryParse(field.Required.ToString(), out isCheck);
+                    itemfield.SystemFieled = bool.TryParse(field.SystemField.ToString(), out isCheck);
+                    if (itemfield.Mandatory)
+                        itemfield.ToUpdate = true;
+                    itemfield.ItemType = testSetfieldInRestSyntax.ToString();
+                    itemfield.Type = field.DataType;
+
+                    //if ((field.ListId != null) && (field.ListId != string.Empty) && (field.FieldValues != null) && (field.FieldValues.Count > 0))
+                    //{
+                    //    foreach (string value in field.AllowedValues)
+                    //    {
+                    //        itemfield.PossibleValues.Add(value);
+                    //    }
+                    //}
+
+                    if (itemfield.PossibleValues.Count > 0)
+                    {
+                        itemfield.SelectedValue = field.DefaultValue;
+                    }
+                    else
+                    {
+                        // itemfield.SelectedValue = "NA";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)